src/main/java/de/unistuttgart/towercrushbackend/data/websockets/Purpose.java

```diff
@@ -1,5 +1,5 @@
 package de.unistuttgart.towercrushbackend.data.websockets;
 
 public enum Purpose {
-    CHAT_MESSAGE,JOIN_TEAM_MESSAGE
+    CHAT_MESSAGE, JOIN_TEAM_MESSAGE, JOIN_LOBBY_MESSAGE, DEVELOPER_MESSAGE
```

Comment:
```suggestion
    CHAT_MESSAGE,
    DEVELOPER_MESSAGE,
    JOIN_LOBBY_MESSAGE,
    JOIN_TEAM_MESSAGE
```
This list will become quite long, so I prefer having one value per line, in alphabetical order.

Reply:
hm idk if the autoformatter will just change it again, but we can try